src/Widgets/DisplayWidget.vala

```diff
@@ -70,7 +70,7 @@ public class BluetoothIndicator.Widgets.DisplayWidget : Gtk.Spinner {
 
         if (state) {
             style_context.remove_class ("disabled");
-            context = _("Middle-click to disable bluetooth");
+            context = _("Middle-click to turn off Bluetooth");
```

Comment:
This is idiomatic, but sounds better to me in English, at least. And you can easily replace `Bluetooth` with a pronoun and it still reads correctly, i.e. `Turn it off`.

```suggestion
            context = _("Middle-click to turn Bluetooth off");
```